### PR TITLE
MC-15617: add docs

### PR DESCRIPTION
**Pull request by dpmurray21**

### CODE REVIEW

**Review by razine-bensari — APPROVED** [review]
looks good senior 🇲🇽

File: source/includes/stackpath/_custom_rules.md [hunk 1]
```diff
@@ -244,6 +244,55 @@ Optional | &nbsp;
 `httpMethods`<br/>*Array[string]* | List of HTTP methods that the rule will apply to. Only for rule of type `REQUEST_RATE`.
 `ipAddresses`<br/>*Array[string]* | List of IP addresses that the rule will apply to. Only for rule of type `REQUEST_RATE`.
 
+<!-- CREATE A CUSTOM RULE BASED ON A WAF REQUEST -->
+
+##### Create a WAF rule based on a request made to an existing site
+
+```
+GET "https://cloudmc_endpoint/api/v1/services/{serviceCode}/{environment}/wafrequests/{wafrequestId}?operation=create_rule&siteId={siteId}"
```
Comment (m-orfanos) [inline source/includes/stackpath/_custom_rules.md]: `POST`?

**Review by m-orfanos — APPROVED** [review]
one small comment, LGTM!

File: source/includes/stackpath/_custom_rules.md (hunk 1 above)
Comment (dpmurray21) [reply source/includes/stackpath/_custom_rules.md]: Yes good catch, fixing now

**Pull request merged**